new/helpers/command.ts

```diff
@@ -68,7 +68,7 @@ export const command: Command = (topic, cmd, stdout, stderr) => (env) =>
           )}`
         );
         reject();
-        process.exit(code);
+        process.exit(code || undefined);
```

Comment:
I'd exit with `1` by default. Exiting with `undefined` turns the script green in the end - also on CI.

Reply:
will do